Sources/Mocker/Mock.swift

```diff
@@ -34,8 +34,8 @@ public struct Mock: Equatable {
 
     public typealias OnRequest = (_ request: URLRequest, _ httpBodyArguments: [String: Any]?) -> Void
 
-    /// The type of the data which is returned.
-    public let dataType: DataType
```

Comment:
Regarding this being a breaking change, do you feel we can work around this? E.g. by deprecating `dataType`, and introducing a new `returnedDataType` (or other name) that is optional.

Reply:
Yes, you're right. It is a breaking change, even though I thought "why would someone read this property?" ;-). (we just don't know how the mocks are used)

I'll make it non-breaking this evening or tomorrow....

Reply:
Yeah, welcome to maintaining frameworks... 😆

Reply:
Like Murphy's Law but different: any API that can be used, will be used.

Reply:
I suggest to use the name `contentType`. 🤓 (When I first used Mocker, I had wondered why the parameter is called `dataType`.)

Reply:
Perfect!